game-core/src/main/java/games/strategy/engine/random/PropertiesDiceRoller.java

```diff
@@ -131,19 +131,13 @@ public String postRequest(final int max, final int numDice, final String subject
         HttpClientBuilder.create().setRedirectStrategy(new AdvancedRedirectStrategy()).build()) {
       final HttpPost httpPost = new HttpPost(props.getProperty("path"));
       final List<NameValuePair> params = ImmutableList.of(
-          new BasicNameValuePair("numdice", "" + numDice),
-          new BasicNameValuePair("numsides", "" + max),
-          new BasicNameValuePair("modroll", "No"),
-          new BasicNameValuePair("numroll", "" + 1),
+          new BasicNameValuePair("numdice", String.valueOf(numDice)),
+          new BasicNameValuePair("numsides", String.valueOf(max)),
           new BasicNameValuePair("subject", message),
           new BasicNameValuePair("roller", getToAddress()),
-          new BasicNameValuePair("gm", getCcAddress()),
-          new BasicNameValuePair("send", "true"));
+          new BasicNameValuePair("gm", getCcAddress()));
       httpPost.setEntity(new UrlEncodedFormEntity(params, StandardCharsets.UTF_8));
       httpPost.addHeader("User-Agent", "triplea/" + ClientContext.engineVersion());
-      // this is to allow a dice server to allow the user to request the emails for the game
-      // rather than sending out email for each roll
-      httpPost.addHeader("X-Triplea-Game-UUID", gameUuid);
```

Comment:
This feature might have been vaporware based on the "future" adjective in the associated (2008) changelog.  From https://github.com/triplea-game/triplea/commit/5ab9c28010f3ad0e0ba01c5f45427075138fd9eb#diff-e8800259c4985f4a73b0936128034c53:

> \* add game id, to be used in future pbem server (sgb)